Dockerfile

```diff
@@ -60,14 +62,8 @@ RUN cd ${TEMP} \
 && tar -xvf nexus-iq-server-${IQ_SERVER_VERSION}-bundle.tar.gz \
 && mv nexus-iq-server-${IQ_SERVER_VERSION}.jar ${IQ_HOME} \
 && cd ${IQ_HOME} \
-&& rm -rf ${TEMP}
-
-# Add group and user
-RUN groupadd -g 1000 nexus \
-&& adduser -u 997 -d ${IQ_HOME} -c "Nexus IQ user" -g nexus -s /bin/false -r nexus
-
-# Change owner to nexus user
```

Comment:
While the changes here generally work, I'd like to propose a slightly different formatting that allows to preserve the logical blocks along with their commentary (cf. https://docs.docker.com/engine/reference/builder/#format):
https://github.com/sonatype/docker-nexus-iq-server/commit/189523172da892e1eec0e3e3673d712f0973cc45

Reply:
oh, nice: if comments can be kept like this, I'm full +1

Reply:
Would you like to incorporate that technicality into your PR and we merge that then or shall I rather just run with my commit? I don't want to make it more complicated for you but ensure you don't feel your thunder has been stolen.

Reply:
no problem, just merge your commit, it's simpler: I know you well, you didn't steal anything but added value :)